include/CL/sycl/device.hpp

```diff
@@ -247,18 +248,23 @@ class device
 
 template <>
 inline auto device::get_info<info::device::max_work_group_size>() const {
-	return static_cast<size_t>(1024);
+  return size_t { 53 };
 }
 
 
 template <>
 inline auto device::get_info<info::device::max_compute_units>() const {
-	return static_cast<size_t>(1);
+  return size_t { 56 };
 }
-	
+
 template <>
 inline auto device::get_info<info::device::device_type>() const {
-	return info::device_type::cpu;
+  return info::device_type::cpu;
+}
+
+template <>
+inline auto device::get_info<info::device::local_mem_size>() const {
+  return size_t { 32000 };
```

Comment:
It looks like it is just what is lacking to have this branch to land now.
A value test for each defined parameter.